content/blog/2021-02-10-onboarding-resource-roundup/contents.lr

```diff
@@ -0,0 +1,67 @@
+title: InvenioRDM Onboarding Resource Roundup
+---
+author: Sara Gonzales
+---
+body:
+
+Congratulations! Your team has just joined the InvenioRDM open source project and is eager to contribute as much as possible. With a mixture of dedicated developers, resource curators, subject matter experts, and various leadership stakeholders, your team simply needs to know about the available resources and how to get started. Read on!
+
+
+### Developers
+
+Whether you would like to install and customize InvenioRDM, contribute on the main project sprints, or take on development of a module locally to contribute back to the project, there are a few important points of entry to participation:
+
+- [Roadmap](https://inveniosoftware.org/products/rdm/roadmap/): Familiarize yourself with the overall project timeline and goals. Stable releases will be made through Summer 2021 on a monthly/bi-monthly basis
```

Comment:
Just "monthly-basis" is good enough I think. It's a more stable commitment (less wishy-washy) and we are upholding it.

Reply:
Agreed - I'll make this update.